Revit_oM/_Properties/AssemblyInfo.cs

```diff
@@ -1,6 +1,6 @@
 /*
  * This file is part of the Buildings and Habitats object Model (BHoM)
- * Copyright (c) 2015 - 2018, the respective contributors. All rights reserved.
+ * Copyright (c) 2015 - 2019, the respective contributors. All rights reserved.
```

Comment:
This should probably be reverted if the file is no longer being changed?

Reply:
Since the 3.0 has been introduced in 2019, I guess we are safe changing 2018 to 2019 as a default in case of all `AssemblyInfo` files.

Reply:
Good point!